src/SFA.DAS.ReferenceData.CharityImport.WebJob/Updater/CharityImporter.cs

```diff
@@ -37,77 +39,54 @@ public async Task RunUpdate()
             var lastImport = await _charityRepository.GetLastCharityDataImport();
             
             // start at May 2017 which will import June 2017 when no previous import exists
-            var importInfo = await SearchForDownload(lastImport != null 
+            var importInfo = await DownloadPublicCharityFile(lastImport != null 
```

Comment:
can we change to not physically download the zip to the filesystem and instead return a stream, like here: https://github.com/SkillsFundingAgency/das-courses-api/blob/580b4f1ef42d23a5bdc6f33725815efaee3b4d97/src/SFA.DAS.Courses.Application/CoursesImport/Services/LarsImportStaging.cs#L44